sklearn/metrics/classification.py

```diff
@@ -1589,11 +1594,17 @@ def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None):
     The logarithm used is the natural logarithm (base-e).
     """
     lb = LabelBinarizer()
-    T = lb.fit_transform(y_true)
+    lb.fit(labels) if labels is not None else lb.fit(y_true)
+    if labels is None and len(lb.classes_) == 1:
+        raise ValueError('y_true has only one label,'
+        'maybe get error log loss, should use labels option')
```

Comment:
This error message should be reworded as `y_true has only one label, provide the true labels explicitly through the labels argument` (or better)

P.S : English isn't my first language.